src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -593,6 +593,24 @@ public with sharing class STG_SettingsManager_TEST {
         system.assertEquals(15, UTIL_CustomSettingsFacade.getOrgRecurringDonationsSettings().npe03__Opportunity_Forecast_Months__c);
     }
 
+    /*********************************************************************************************************
+    * @description Tests the Recurring Donations panel when RD2 is enabled
+    * @return void
+    */
+    public static testMethod void checkCloseActionsContainsSelectOptionsWhenRD2Enabled() {
```

Comment:
Do we have a test to verify close options (actions) for legacy RDs (enhanced RDs are not enabled)?

Reply:
@RadSFDeveloper , I added another test for legacy RDs too.